docs/Installation.rst

```diff
@@ -53,8 +55,9 @@ On Ubuntu 14.10, one will need to run the following script to satisfy all depend
 
 .. sourcecode:: bash
 
-    sudo apt-get install build-essential fpc postgresql postgresql-client \
-         gettext python2.7 iso-codes shared-mime-info stl-manual cgroup-lite
+    sudo apt-get install build-essential openjdk-8-jre openjdk-8-jdk fpc \
```

Comment:
However, I think it's OK to put it with a clear "optional" notice, so that the majority of users (who may not even need Java support) don't end up installing a PPA without needing it